memory_model/src/guest_memory.rs

```diff
@@ -424,6 +451,52 @@ impl GuestMemory {
         }
         Err(Error::InvalidGuestAddress(guest_addr))
     }
+
+    /// Read the whole content from one or more MemoryRegions
+    fn do_in_region_full<F>(&self, guest_addr: GuestAddress, size: usize, mut cb: F) -> Result<()>
+    where
+        F: FnMut(&MemoryMapping, usize, usize) -> Result<()>,
+    {
+        // Fast path: content doesn't cross MemoryRegion boundary
```

Comment:
Fast path is the same code as `do_in_region()`. Why not call that here directly and in case of `Err` move to the slow-path.

Would help de-duplicate code.

Reply:
do_in_region_full() has been removed.